apis/iplocation.py

```diff
@@ -65,14 +65,15 @@ def get_ip_from_domain(self, domain: str) -> ipaddress.IPv4Address:
     def is_in_safe_country(self, country_code: str) -> bool:
         return country_code not in self.UNSAFE_COUNTRY_CODES
 
-    async def is_site_safe(self, domain: str) -> bool:
+    async def is_in_db(self, domain: str) -> bool:
         try:
             ip_addr = self.get_ip_from_domain(domain)
             location_data = await self.get_location_info(ip_addr)
             _LOGGER.debug(f'Unsafe country codes are {self.UNSAFE_COUNTRY_CODES}, and the current country code is {location_data.countryCode}')
-            return self.is_in_safe_country(location_data.countryCode.lower())
+            return not self.is_in_safe_country(location_data.countryCode.lower())
```

Comment:
`is_in_safe_country` checks if the country code is not in the unsafe list, notting that will allow only the unsafe countries